### ISSUE TITLE
ImageMagick write to ASP.NET OutputStream

### ISSUE THREAD

**Issue by BionicCa** [dlemstra/Magick.NET#56]
I just updated to Magick.Net 7.0.5.900 from version 7.0.2.900 and getting this Exception: System.ArgumentException: The stream should be readable.

Example Code: 
```
MagickImage image = new MagickImage(imagePath);
image.Crop(0, 0, 150, 150);
image.Write(HttpContext.Current.Response.OutputStream);
```

This Code runs perfectly with the older Version. 
I'm not exactly sure why a stream should be readable when only a writing permission should do that Job. Any Suggestions how I can write the Image directly to Response Output Stream?

**Comment by dlemstra** [comment]
I made some changes to Magick.NET to read/write streams more efficient. I cannot remember why I required the Write method to supply a readable stream. The code in ImageMagick even has `asserts` for this. The only thing that would require this would be an encoder that would also read from the bytes that it has written. But I am pretty sure that this won't happen. I will investigate it this weekend and publish a new release as soon as possible to resolve your issue.
